test/execute.js

```diff
@@ -529,13 +529,60 @@ describe('execute', () => {
         headers: {
           head: 'justTheHead',
         },
-        body: {
+        body: JSON.stringify({
           json: 'rulez'
-        }
+        })
       })
     })
   })
 
+  it('should pass through the parameters object as the value if parameter is in the body', function () {
```

Comment:
I don't understand what "pass through" here means. And what value does this test add compared to existing one such as "should build a request for all given field"?